de.nagstamon.nagstamon.yml

```diff
@@ -0,0 +1,111 @@
+app-id: de.nagstamon.nagstamon
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: nagstamon
+rename-desktop-file: nagstamon.desktop
+rename-icon: nagstamon
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=.nagstamon
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --filesystem=xdg-run/keyring
+  - --filesystem=home
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - name: libsecret
+    buildsystem: meson
+    config-opts:
+      - -Dgcrypt=false
+      - -Dgtk_doc=false
+      - -Dintrospection=false
+      - -Dmanpage=false
+      - -Dvapi=false
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/libsecret/-/archive/0.20.5/libsecret-0.20.5.tar.gz
+        sha256: b33b9542222ea8866f6ff2d31c0ad373877c2277db546ca00cc7fdda9cbab1c3
+  - name: krb5
+    subdir: src
+    config-opts:
+      - --localstatedir=/var/lib
+      - --sbindir=${FLATPAK_DEST}/bin
+      - --disable-static
+      - --disable-rpath
+    cleanup:
+      - /bin
+      - /share/et
+      - /share/examples
+      - /share/man
+    sources:
+      - type: archive
+        url: https://github.com/krb5/krb5/archive/refs/tags/krb5-1.21.2-final.tar.gz
+        sha256: 5827fa39ead0e7f7b8a158799eb5d50b194129e81b378892caedcafe50c5c195
+      - type: shell
+        dest: src
+        commands:
+          - autoreconf -si
+  - name: dbus-run-session
+    buildsystem: autotools
+    cleanup: ['*']
+    sources:
+      - type: archive
+        url: https://gitlab.freedesktop.org/dbus/dbus/-/archive/dbus-1.14.10/dbus-dbus-1.14.10.tar.gz
+        sha256: 5fedcd6c17fbc62646af224e1aab11a334c463e7e7d89163a670ed24e7ab241c
+        x-checker-data:
+          type: anitya
+          project-id: 5356
+          url-template: https://gitlab.freedesktop.org/dbus/dbus/-/archive/dbus-$version/dbus-dbus-$version.tar.gz
+  - name: dbus-python
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz
+        sha256: ad67819308618b5069537be237f8e68ca1c7fcc95ee4a121fe6845b1418248f8
+        x-checker-data:
+          type: anitya
+          project-id: 402
+          url-template: https://dbus.freedesktop.org/releases/dbus-python/dbus-python-$version.tar.gz
+
+  - modules/python3-cryptography.yml
+  - modules/python3-lxml.yml
+  - modules/python3-beautifulsoup4.yml
+  - modules/python3-keyring.yml
+  - modules/python3-requests.yml
+  - modules/python3-psutil.yml
+  - modules/python3-dateutil.yml
+  - modules/python3-requests-kerberos.yml
```

Comment:
These python manifests should be combined into one. It doesn't help if your dep tree keeeps growing and each small module gets a new manifest.

Reply:
So it is a better practice to put them all into one?

Reply:
Yes, pass all to flatpak-pip-generator, then pip can point out dependency conflicts too.

Reply:
They are all integrated in the manifest.

Reply:
> They are all integrated in the manifest.

I meant a separate manifest that has all of them if you pass `flatpak-pip-generator lxml dateutil ... ` you get a `python3--modules.yml` -  I'm talking about that.

If you add them in the main manifest you cannot easily update them. If they are in a separate one you can just run `flatpak-pip-generator` again and copy-paste the file.

Reply:
Sorry. So I create a modules/python-modules.yaml file containing all modules prior being single files?

Reply:
Yes, it can be in the same directory as the manifest, it doesn't need to be in a modules subdir.

Reply:
Added python-modules.yml now.